backend/src/main/java/com/bakdata/conquery/models/query/resultinfo/printers/StringResultPrinters.java

```diff
@@ -18,6 +19,39 @@
 @ToString
 public class StringResultPrinters extends PrinterFactory {
 
+	/**
+	 * This is equivalent to `-∞`. For some reason ∞/infinity does not get rendered in WINDOWS-1252. This way however it works.
+	 * The code is taken from <a href="https://www.ee.ucl.ac.uk/mflanaga/java/HTMLandASCIItableWin.html">this table</a>.
+	 */
+	public static final String NEGATIVE_INF_W1252 = "-" + new String(new byte[]{0x22, 0x1E}, 0, 1, Charset.forName("WINDOWS-1252"));
+	/**
+	 * This is equivalent to `+∞`. For some reason ∞/infinity does not get rendered in WINDOWS-1252. This way however it works.
+	 * The code is taken from <a href="https://www.ee.ucl.ac.uk/mflanaga/java/HTMLandASCIItableWin.html">this table</a>.
+	 */
+	public static final String POSITIVE_INF_W1252 = "+" + new String(new byte[]{0x22, 0x1E}, 0, 1, Charset.forName("WINDOWS-1252"));
+
+
+	public static final String POSITIVE_INF_DEFAULT = "+∞";
+	public static final String NEGATIVE_INF_DEFAULT = "-∞";
+	private final String negativeInf;
+	private final String positiveInf;
+
+	// Public for tests only.
+	public StringResultPrinters(String negativeInf, String positiveInf) {
+		this.negativeInf = negativeInf;
+		this.positiveInf = positiveInf;
+	}
+
+	/**
+	 * This circumvents a bug in the translation of infinity when using WINDOWS-1252 Charset.
+	 */
+	public static StringResultPrinters forCharset(Charset charset) {
+		if ("windows-1252".equals(charset.name()) || charset.aliases().contains("windows-1252")) {
```

Comment:
Hier vielleicht einfach mit einer static Version von MS1252 arbeiten oder
```suggestion
		if (charset.contians(new MS1252())) {
```

Reply:
Ich dachte, das ist nur eine subset geschichte, muss mir mal anschauen was das macht. wäre natürlich etwas chilliger. Aber der Constructor hat in meiner Erinnerung nicht funktioniert - kann aber auch IntelliJ bug sein.